src/PlugWallet/index.ts

```diff
@@ -191,7 +195,17 @@ class PlugWallet {
   };
 
   public getTransactions = async (): Promise<GetTransactionsResponse> => {
-    return getTransactions(this.accountId);
+    const icpTrxs = await getICPTransactions(this.accountId);
+    const xtcTransactions = await getXTCTransactions(this.principal);
```

Comment:
Could these be done in parallel?

Reply:
They actually could, will open a new pr with an improvement for later